src/app/kuma/index.ts

```diff
@@ -13,6 +13,9 @@ export const TOKENS = {
 }
 export const services = (app: Record<string, Token>): ServiceDefinition[] => {
   return [
+    [app.storagePrefix, {
+      service: () => 'kumahq.kuma-gui',
+    }],
```

Comment:
Sets the localStorage key to use a `kuma.kuma-gui` prefix. So each route saves its data in `kuma.kuma-gui:name-of-route`. We'll need to be able to alter this prefix depending on the application, hence we've exposed it as a dependency injection token (also see above comment)